grakn-test/src/test/java/ai/grakn/test/graql/analytics/ScalingTestIT.java

```diff
@@ -301,7 +304,8 @@ public void testLargeDegreeMutationResultsInReadableGraphIT() throws Exception {
         addEdges(keyspace, MAX_SIZE);
 
         Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().showImplicitConcepts(true);
-        Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().graql().compute().degree().persist().execute();
+        //TODO: Remove persistence
```

Comment:
Scaling test heavily relies on persisting. I'd like to talk to @sheldonkhall next week about what we wanna do with the scaling test.